tests/shared/models/selection-state.spec.js

```diff
@@ -890,39 +890,6 @@ describe('Model: selectionState', function () {
         m.chai.expect(selectionState.hasImage()).to.be.false
       })
     })
-
-    describe('given the preserveImage option', function () {
```

Comment:
So is this functionality not needed anymore?

Reply:
The `preserveImage` option is only used in one instance – https://github.com/resin-io/etcher/pull/2043/files#diff-be4cad3a825aa7d1fd460dd90806eda2R48 – so I think it makes sense to just move the logic there where it's used instead.

Reply:
Alright. In that case, can we move these test cases to a `test/gui/pages/finish.spec.js` file?

Reply:
Ah, but its going to be tricky to test that given `$state` and `analytics` :( @jhermsmeier What do you think?

Reply:
Maybe we can sinon spy on `selectionState.deselectImage` and `selectionState.deselectDrive`?

Reply:
I'd like to avoid spies as most as possible. If we really want to test this, then we should split `finish.restart()` into two functions and test the one that deselects stuff separately, but then its a simple enough logic that I think we should instead test with something like Spectron.

Reply:
@Shou wdyt about what @jviotti said?

Reply:
@jhermsmeier @jviotti what if we just rename the existing test block from `.clear()` to `.deselectImage() and .deselectDrive()` given the intention is to test that these two work together?

Reply:
Sounds good to me